src/components/Icon/__tests__/all-icons-tests.jest.ts

```diff
@@ -1,17 +1,21 @@
 import { exposeIcons } from "../../../utils/icon-utils";
 import * as allIcons from "../Icons";
 
-function arrayLast(arr) {
+function arrayLast(arr: Array<string>) {
   if (!arr) return null;
   return arr[arr.length - 1];
 }
 
 describe("All icons", () => {
   it("should contains all icons", () => {
     const allIconFiles = exposeIcons();
-    const missingIcons = [];
+    const missingIcons: Array<string> = [];
     Object.keys(allIconFiles).forEach(iconPath => {
       const iconName = arrayLast(iconPath.split("/"));
+      /**
+       * TS thinks allIcons is an array but its an object which contains all icons and that's why it's work when iconName is string
+       */
+      // @ts-ignore
```

Comment:
The convention is to write the reason on the same line as the `@ts-ignore` directive.

Reply:
This is strange, TS should figure out that `import * as` returns an object. What's the exact error here?

Reply:
Yes I did not understand the error... I fixed the comment. please tell me if it is not enough.